### DISCUSSION TITLE
generated canny map is not aligned with the image.

### DISCUSSION THREAD

**Discussion by MrityunjayBhardwaj** [Mikubill/sd-webui-controlnet#533] [Q&A]
input image mask:-
![product_33_pc(1)](https://user-images.githubusercontent.com/44331829/223622087-75526883-7fa5-409a-ba79-9f580519a795.png)

generated canny map:
![temp_cnet_img(1)](https://user-images.githubusercontent.com/44331829/223621968-d2bc2c90-7cc5-4b15-8f5f-1d736c17883d.png)

is there a way to upload a custom canny map as an input to control net?

**Comment by missionfloyd** [comment]
> is there a way to upload a custom canny map as an input to control net?

Upload it like any other image and select "none" for preprocessor.
